packages/protocol/test/identity/odispayments.ts

```diff
@@ -0,0 +1,146 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import {
+  assertEqualBN,
+  assertLogMatches2,
+  assertRevert,
+  assumeOwnership,
+} from '@celo/protocol/lib/test-utils'
+import { getDeployedProxiedContract } from '@celo/protocol/lib/web3-utils'
+import { fixed1 } from '@celo/utils/src/fixidity'
+import {
+  FreezerContract,
+  FreezerInstance,
+  OdisPaymentsContract,
+  OdisPaymentsInstance,
+  RegistryInstance,
+  StableTokenContract,
+  StableTokenInstance,
+} from 'types'
+
+const Freezer: FreezerContract = artifacts.require('Freezer')
+const OdisPayments: OdisPaymentsContract = artifacts.require('OdisPayments')
+const StableTokenCUSD: StableTokenContract = artifacts.require('StableToken')
+
+const SECONDS_IN_A_DAY = 60 * 60 * 24
+
+contract('OdisPayments', (accounts: string[]) => {
+  let freezer: FreezerInstance
+  let odisPayments: OdisPaymentsInstance
+  let registry: RegistryInstance
+  let stableTokenCUSD: StableTokenInstance
+
+  const owner = accounts[0]
+  const sender = accounts[1]
+  const startingBalanceCUSD = 1000
+
+  before(async () => {
+    // Mocking Registry.sol when using UsingRegistryV2.sol
+    registry = await getDeployedProxiedContract('Registry', artifacts)
```

Comment:
By convention, in the monorepo tests we don't rely on contracts deployed during the Truffle migrations, instead deploying any necessary contracts with `<Contract>.new` in a `beforeEach`.

Reply:
Ooh, I see, this would break with UsingRegistryV2, given it relies on Registry being predeployed at `0xce10`.

Reply:
exactly!